assets/app/views/_pods.html

```diff
@@ -1,11 +1,14 @@
 <!--
-  Expects the following variables:
-  pods - a hash of pod objects by id
+Expects the following variables:
+pods - a hash of pod objects by id
 -->
 <div class="component-block pod-container">
-  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod">
+  <div class="animate-repeat pod pod-{{pod.status.phase.toLowerCase()}} osc-object"  ng-repeat="pod in pods track by (pod | uid)" kind="Pod" resource="pod" ng-init="showWarning = (pod | isTroubledPod)" ng-class="{'pod-warning': showWarning}">
     <div class="pod-text">
-      <strong>{{pod.status.phase}}</strong><div ng-if="pod.status.podIP">{{pod.status.podIP}}</div><div ng-if="!pod.status.podIP">allocating IP...</div>
+      <!-- Add a tooltip only if showing a pod warning -->
+      <strong ng-if="showWarning" data-toggle="tooltip" title="Check pod and container statuses for problems">{{pod.status.phase}}</strong>
```

Comment:
@liggitt So remove the tooltip?

I don't like showing a warning style, but giving no hint at all as to what might be wrong. Showing a detailed message is best, but at least telling them it has something to do with the container statuses users points the user in a direction.